internal/cephfs/volumeoptions.go

```diff
@@ -48,6 +48,32 @@ type volumeOptions struct {
 	FuseMountOptions    string `json:"fuseMountOptions"`
 	SubvolumeGroup      string
 	Features            []string
+
+	// conn is a connection to the Ceph cluster obtained from a ConnPool
+	conn *util.ClusterConnection
+}
+
+// Connect an CephFS volume to the Ceph cluster.
+func (vo *volumeOptions) Connect(cr *util.Credentials) error {
+	if vo.conn != nil {
+		return nil
+	}
+
+	conn := &util.ClusterConnection{}
+	if err := conn.Connect(vo.Monitors, cr); err != nil {
+		return err
+	}
+
+	vo.conn = conn
+	return nil
+}
+
+// Destroy cleans up the CephFS volume and closes the connection to the Ceph
+// cluster in case one was setup.
+func (vo *volumeOptions) Destroy() {
```

Comment:
will destroy cleanup the volume? its just closes the connect right.

Reply:
it destroys resources used by the `volumeOptions` instance, so it depends what you mean with 'cleanup' I guess